package.json

```diff
@@ -18,18 +18,19 @@
   },
   "homepage": "https://github.com/firstandthird/parse-dir",
   "devDependencies": {
-    "eslint": "^1.10.3",
-    "eslint-config-airbnb": "^5.0.1",
-    "eslint-config-firstandthird": "^1.4.0",
-    "grunt": "^0.4.5",
-    "grunt-contrib-jshint": "^0.10.0",
-    "grunt-contrib-watch": "^0.6.1",
-    "grunt-mocha-test": "^0.10.2",
-    "load-grunt-config": "^0.9.2"
+    "chai": "^3.5.0",
+    "eslint": "^2.9.0",
+    "eslint-config-airbnb": "^8.0.0",
+    "eslint-config-firstandthird": "^2.0.0",
+    "eslint-plugin-import": "^1.5.0",
+    "eslint-plugin-jsx-a11y": "^1.0.2",
+    "eslint-plugin-react": "^5.1.1",
+    "mocha": "^2.5.3"
   },
   "dependencies": {
-    "glob": "^4.0.0",
     "async": "^0.9.0",
+    "coffee-script": "^1.10.0",
```

Comment:
That dependency upgrades the 'require' keyword, so that node can require coffeescript modules just like they were js modules.  Removing it breaks parse-dir's support for coffeescript.